internal/models/results.go

```diff
@@ -28,13 +35,16 @@ type PickleStepResult struct {
 	PickleStepID string
 
 	Def *StepDefinition
+
+	Attachments []*PickleAttachment
```

Comment:
I'd change this to slice of values if there is no specific reason for a slice of pointers.